services/rds/rds.go

```
@@ -219,6 +314,40 @@ func (d *dedicatedDBAdapter) bindDBToApp(i *RDSInstance, password string) (map[s
 	return i.getCredentials(password)
 }
 
+// search out all the parameter groups that we created and try to clean them up
+func cleanupCustomParameterGroups(svc *rds.RDS) {
+	input := &rds.DescribeDBParameterGroupsInput{}
+	err := svc.DescribeDBParameterGroupsPages(input,
+		func(pgroups *rds.DescribeDBParameterGroupsOutput, lastPage bool) bool {
+			// If the pgroup matches the prefix, then try to delete it.
+			// If it's in use, it will fail, so ignore that.
+			for _, pgroup := range pgroups.DBParameterGroups {
+				if matched, _ := regexp.Match("^"+PgroupPrefix, []byte(*pgroup.DBParameterGroupName)); matched {
```

Can you add error handling here? I don't want to swallow errors.

---

I guess we could log an error.  Good call!  Will have that in shortly.

---

done!